Src/WindowsServer/WindowsServer.Shared/Implementation/WindowsServerEventSource.cs

```diff
@@ -326,6 +326,29 @@ public void AzureInstanceMetadataFailureSettingDefaultPayload(string exception,
                 this.ApplicationName);
         }
 
+        [Event(33,
+            Message = "Failed to obtain Azure App Services environment variable '{0}'. Exception raised: {1}",
+            Level = EventLevel.Verbose)]
+        public void HeartbeatPropertyAquisitionFailed(string envVarName, string exceptionStr, string appDomainName = "Incorrect")
+        {
+            this.WriteEvent(
+                33,
+                envVarName ?? "unknown",
+                exceptionStr ?? "unknown-exception",
+                this.ApplicationName);
+        }
+
+        [Event(34,
+            Message = "Could not gain access to the Heartbeat Manager instance during initialization. Exception raised: {0}",
+            Level = EventLevel.Verbose)]
```

Comment:
Make it warning at least. This error is very unlikely to happen. It indicates that something is broken in SDK. So we want to have it at least at warning level.

Reply:
Agreed. I've gone over the other calls and altered their levels (including some I had set previously for the Azure IMS data as well).